end2end/tests/adminPanelLinks.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach('Go to Admin Page', async ({ page }) => {
+  await page.goto('https://host.docker.internal/Test_Request_Portal/');
+  await page.getByRole('link', { name: 'Admin Panel' }).click();
+});
+
+test('Verify Admin Page', async ({ page }) => {
+  await expect(page.locator('#bodyarea')).toContainText('Get Help');
+});
+
+test('Go to User Access Page', async ({ page }) => {
+  await page.getByRole('button', { name: ' User Access Groups Modify' }).click();
```

Comment:
remove all the special characters inserted on the locators

Reply:
Done. Hopefully, I committed the changes correctly